checker/jtreg/sortwarnings/OrderOfCheckers.out

```diff
@@ -0,0 +1,19 @@
+OrderOfCheckers.java:15:60: compiler.err.proc.messager: [[all, value]:assignment.type.incompatible] incompatible types in assignment.
+found   : @UnknownVal int @UnknownVal []
```

Comment:
I want to use `-AshowSuppressWarningKeys` so that this tests which checker is issuing the error. If I use `-Anomsgtext`, the keys aren't printed.  I documented this in Main.java.